pkgs/by-name/ar/ares/package.nix

```diff
@@ -2,58 +2,64 @@
   lib,
   SDL2,
   alsa-lib,
-  autoPatchelfHook,
+  apple-sdk_12,
+  cmake,
   fetchFromGitHub,
   gtk3,
   gtksourceview3,
+  gnumake,
   libGL,
   libGLU,
   libX11,
   libXv,
   libao,
-  libicns,
   libpulseaudio,
   librashader,
+  ninja,
+  moltenvk,
   openal,
   pkg-config,
   stdenv,
+  stdenvNoCC,
   udev,
   vulkan-loader,
-  which,
   wrapGAppsHook3,
+  zlib,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "ares";
-  version = "141";
+  version = "142";
 
   src = fetchFromGitHub {
     owner = "ares-emulator";
     repo = "ares";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-iNcoNdGw/DfYc9tsOGsPYoZLhVwNzJe8bVotx6Rl0j4=";
+    hash = "sha256-D/6r4f6dAIsP9JglAmzxlCwV5NoOuw2bZMZ2T1oJ9Jc";
   };
 
-  patches = [
-    ./patches/001-dont-rebuild-on-install.patch
-  ];
-
   nativeBuildInputs =
     [
-      autoPatchelfHook
+      cmake
+      gnumake
```

Comment:
I think `gnumake` is already included in the standard build environment. It was only needed previously because you were using `stdenvNoCC` for the shader derivation.

```suggestion
```